core/lib/json_api_client_extension/resource_extension.rb

```diff
@@ -0,0 +1,23 @@
+# MonkeyPatching json_api_client, see: https://github.com/chingor13/json_api_client/pull/263
```

Comment:
Could we fork, and point to the gem instead ? Or is it better to monkey patch ? @ouranos what is your take on this ?